src/daily_output.ts

```diff
@@ -66,13 +68,37 @@ export class DailyOutput {
 	 * - group effort rewards, and
 	 * - streak rewards
 	 */
-	private finishDataAggregation(): void {
+	private async finishDataAggregation(): Promise<void> {
+		// check for previous unbundled bundles
+		const prevNonUnbundledBundles: { txId: string; tip: number }[] = existsSync('non_unbundled_bundles.json')
+			? JSON.parse(readFileSync('non_unbundled_bundles.json').toString())
+			: [];
+		const unbundledTransactions = prevNonUnbundledBundles.length
+			? await getBundledTransactions(prevNonUnbundledBundles.map(({ txId }) => txId))
```

Comment:
I think you'll want `feedGQLData` to become a promise once again here to do this. Sorry for having you refactor that, seems it was meant to be a promise

Reply:
You could also consider a class that handles tracking unbundled bundles and uses `fs`

Reply:
79807f400c121c79fbbf742f737d4413bd09162d